caffe2/python/operator_test/rnn_cell_test.py

```diff
@@ -126,11 +132,11 @@ def layer_norm_lstm_reference(
         )
 
         hidden_t, cell_t = lstm_unit(
```

Comment:
May be we should keep kwargs usage, it is less error prone. it is also related to the previous comment

Reply:
same answer as above - my hand is forced by how the test machinery uses this function